lib/config/presets/internal/index.spec.ts

```diff
@@ -43,4 +43,15 @@ describe('config/presets/internal/index', () => {
       }
     }
   }
+
+  it('internal presets should not contain handlebars', () => {
+    Object.entries(internal.groups)
```

Comment:
there are a lot more international presets to check. those tests should be on a separate PR

Reply:
> there are a lot more international presets to check

This tests iterates over all presets defined in [/lib/config/presets/internal/index.ts#L20](https://github.com/lstoeferle/renovate/blob/f25d6559276f8ad78328a06e35be74d8d2bfd4be/lib/config/presets/internal/index.ts#L20). As far as I can see the object contains ALL internally defined presets.

> those tests should be on a separate PR

Will do :+1:

Reply:
Moved the test to a [separate PR](https://github.com/renovatebot/renovate/pull/28336)